contrib/packs/actions/pack_mgmt/download.py

```diff
@@ -100,20 +100,27 @@ def _clone_repo(temp_dir, repo_url, verifyssl=True, ref='master'):
         # because we want the user to work with the repo in the
         # future.
         repo = Repo.clone_from(repo_url, temp_dir)
+        active_branch = repo.active_branch
```

Comment:
Looks fine on the glance, but I'm slightly worried there might be cases when `active_branch` may not point to remote's default branch, but rather just be local checkout or something else entirely.